DataFormats/RecoCandidate/BuildFile.xml

```diff
@@ -8,6 +8,11 @@
 <use name="DataFormats/HcalRecHit"/>
 <use name="clhep"/>
 <use name="SimDataFormats/TrackingAnalysis"/>
+<use name="DataFormats/EcalRecHit"/>
```

Comment:
"DataFormats/HcalRecHit", "DataFormats/EcalRecHit", "DataFormats/HcalDetId" are only included inside DataFormats/RecoCandidate/src/classes.h , but actually not needed (and not even included in the classes_def.xml file).
I think one could rather remove them from classes.h, and from this BuildFile